### PR TITLE
Add Simpad Package

### PR DESCRIPTION
**Pull request by frju365**

### CODE REVIEW

**Comment by frju365** [comment]
oh... yes. Ok.

**Comment by frju365** [comment]
fixed !

File: community.json [hunk 1]
```diff
@@ -897,6 +897,12 @@
         "state": "inprogress",
         "url": "https://github.com/YunoHost-Apps/sickrage_ynh"
     },
+    "simpad": {
+        "branch": "master",
+        "revision": "77e757061a33d3fee6913ca7f4230d5a26198976",
+        "state": "working",
+        "url": "https://github.com/YunoHost-Apps/simpad_ynh"
```
Comment (JimboJoe) [inline community.json]: Hmm you're missing the https://... prefix !

Comment (JimboJoe) [inline community.json]: ... and you copied from here which isn't right as well.

Comment (frju365) [inline community.json]: where ?

**Pull request merged**